### PR TITLE
Set up application structure for client(User)

### PR DESCRIPTION
**Pull request by viktorsommar**
### User Story
```
As a development
In order to be able to start adding features
We would like to have a basic application structure
```
## Changes proposed in this pull request:
* Scaffolding react application
## What I have learned working on this feature:
* To scaffold
## Packages/Gems added
- react

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+      content="Web site created using create-react-app"
+    />
+    <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
+    <!--
+      manifest.json provides metadata used when your web app is installed on a
+      user's mobile device or desktop. See https://developers.google.com/web/fundamentals/web-app-manifest/
+    -->
+    <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
+    <!--
+      Notice the use of %PUBLIC_URL% in the tags above.
```
Comment (Odeane) [inline public/index.html]: please clean up comments form your index.html

File: public/manifest.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
+{
+  "short_name": "React App",
```
Comment (Odeane) [inline public/manifest.json:2]: please clean up

**Review by Cma-B — APPROVED** [review]
good job !

File: src/App.jsx [hunk 3]
```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import logo from './logo.svg';
+import './App.css';
+
+function App() {
+  return (
+    <div className="App">
+      <header className="App-header">
+        <img src={logo} className="App-logo" alt="logo" />
```
Comment (Odeane) [inline src/App.jsx]: we cannot merge react sample logo component to our Development ....so please clean up

File: src/index.js [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+import * as serviceWorker from './serviceWorker';
+
+ReactDOM.render(
+  <React.StrictMode>
+    <App />
+  </React.StrictMode>,
+  document.getElementById('root')
+);
+
+// If you want your app to work offline and load faster, you can change
```
Comment (Odeane) [inline src/index.js]: please clean up comment

File: src/index.js [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+// unregister() to register() below. Note this comes with some pitfalls.
+// Learn more about service workers: https://bit.ly/CRA-PWA
+serviceWorker.unregister();
```
Comment (Odeane) [inline src/index.js:14]: we also are not using service worker in this project

File: src/App.jsx [hunk 6]
```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+
+function App() {
```
Comment (emtalen) [inline src/App.jsx]: Change this into an arrow function

**Review by Odeane — APPROVED** [review]
You just need to use the ES6 arrow function like Emma said but otherwise it looks great

**Pull request merged**